Add the new MPM to the build system as an alternative to the other MPMs.

Index: httpd-2.4.1/server/mpm/config2.m4
===================================================================
--- httpd-2.4.1.orig/server/mpm/config2.m4
+++ httpd-2.4.1/server/mpm/config2.m4
@@ -1,7 +1,7 @@
 AC_MSG_CHECKING(which MPM to use by default)
 AC_ARG_WITH(mpm,
 APACHE_HELP_STRING(--with-mpm=MPM,Choose the process model for Apache to use by default.
-                          MPM={event|worker|prefork|winnt}
+                          MPM={event|worker|prefork|winnt|itk}
                           This will be statically linked as the only available MPM unless
                           --enable-mpms-shared is also specified.
 ),[
Index: httpd-2.4.1/modules/arch/unix/config5.m4
===================================================================
--- httpd-2.4.1.orig/modules/arch/unix/config5.m4
+++ httpd-2.4.1/modules/arch/unix/config5.m4
@@ -3,6 +3,7 @@ APACHE_MODPATH_INIT(arch/unix)
 
 if ap_mpm_is_enabled "worker" \
    || ap_mpm_is_enabled "event" \
+   || ap_mpm_is_enabled "itk" \
    || ap_mpm_is_enabled "prefork"; then
     unixd_mods_enable=yes
 else
